### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by adrianne-helpscout**
Adding a link to our support address and small re-order to more closely match ios: https://github.com/helpscout/beacon-ios-sdk

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,3 +103,15 @@ Beacon beacon = Beacon.Builder()
...
+## Documentation
+
+* [Beacon Mobile SDK documentation](https://developer.helpscout.com/beacon-2/mobile/)
+* [Android SDK API](https://developer.helpscout.com/beacon-2/android/)
+* [Android SDK API reference](https://developer.helpscout.com/beacon-2/android-api/beacon/index.html)
+
+
+## Support
+
+For issues or questions, contact Help Scout support at help@helpscout.com
```
Comment (aliripe) [inline README.md:117]: 💡 Do we want to provide some sort of template leveraging the `mailto` label?

[help@helpscout.com](mailto:help@helpscout.com?subject=%5BAndroid%20Beacon%20SDK%5D%20Issue%20report&body=Technical%20details%0A%0A-%20Beacon%20version%3A%0A-%20Android%20OS%20version%3A%0A-%20Device%20make%2Fmodel%3A%0A-%20Impacted%20user%20count%3A%0A%0AActual%20outcome%0A%0A%20%20%20%20Beacon%20did%20or%20did%20not...%0A%0AExpected%20outcome%0A%0A%20%20%20%20Beacon%20should...%0A%0ASteps%20to%20reproduce%0A%0A%20%20%20%201.%20%E2%80%A6%0A%20%20%20%202.%20%E2%80%A6%0A%20%20%20%203.%20%E2%80%A6%0A%0AScreenshots%2FScreen%20Recording%0A%0A%20%20%20%20Please%20add%20any%20images%20or%20videos%20to%20support%20your%20report.%0A%0AAdditional%20Information%0A%0A%20%20%20%20Please%20attach%20any%20crash%20logs)

When clicked, it looks like this:
Screen Shot 2022-02-03 at 10 01 34

```suggestion
For issues or questions, contact Help Scout support at [help@helpscout.com](mailto:help@helpscout.com?subject=%5BAndroid%20Beacon%20SDK%5D%20Issue%20report&body=Technical%20details%0A%0A-%20Beacon%20version%3A%0A-%20Android%20OS%20version%3A%0A-%20Device%20make%2Fmodel%3A%0A-%20Impacted%20user%20count%3A%0A%0AActual%20outcome%0A%0A%20%20%20%20Beacon%20did%20or%20did%20not...%0A%0AExpected%20outcome%0A%0A%20%20%20%20Beacon%20should...%0A%0ASteps%20to%20reproduce%0A%0A%20%20%20%201.%20%E2%80%A6%0A%20%20%20%202.%20%E2%80%A6%0A%20%20%20%203.%20%E2%80%A6%0A%0AScreenshots%2FScreen%20Recording%0A%0A%20%20%20%20Please%20add%20any%20images%20or%20videos%20to%20support%20your%20report.%0A%0AAdditional%20Information%0A%0A%20%20%20%20Please%20attach%20any%20crash%20logs)
```

**Review by aliripe — APPROVED** [review]
LGTM with a suggestion! 🙌

File: README.md (hunk 1 above)
Comment (scottyab) [reply README.md:117]: 💯  I was gonna suggestion mailto: but 🏅for the prompts for all that extra info

**Review by scottyab — APPROVED** [review]
👍 agree with Alicia's suggestion to add mailto with prepopulated info

**Pull request merged**